lib/css/components/badges.less

```
@@ -59,6 +59,15 @@
     margin-left: calc((var(--su4) + var(--su1)) * -1);
 }
 
+//  $   BADGE ICON
+//      When we have a badge icon
+//  ---------------------------------------------------------------------------
+.s-badge__icon {
+    .svg-icon {
+        margin-right: var(--su4);
+    }
```

I think we could reuse the code from s-post-summary here so we don't have to re-override it there (and it gives a bit of extra consistency as well)

```suggestion
    gap: 0.3em;
```